backend/czi_hosted/data_common/cache.py

```diff
@@ -155,7 +157,7 @@ def data_adaptor(self, cache_key: str, create_data_lambda: Optional[Callable]=No
                 cache_item = CacheItem()
                 desired_data = cache_item.get(
                     cache_key=cache_key,
-                    create_data_lambda=create_data_lambda,
+                    create_data_lambda=create_data_function,
```

Comment:
any reason you use two different names for the same item/value?   (ie, lambda vs function)?

Reply:
No they should all be the same. I'll update them